Autocoders/MagicDrawCompPlugin/doc/man/man3/gov_nasa_jpl_componentaction_IsfCommandLine.3

```diff
@@ -54,7 +54,7 @@ Class to run MagicDraw plugin from command line
 .PP
 Before running on headless environment, use Xvfb to set up a virtual X11 display server \fC/usr/bin/Xvfb :1 -screen 0 1024x768x24 export DISPLAY=:1 \fP
 .PP
-To run the utility, use the script isfpluginexec\&.sh in the MagicDrawComdPlugin directory 
+To run the utility, use the script isfpluginexec\&.sh in the MagicDrawCompPlugin directory 
```

Comment:
There are a number of different misspellings of this directory name. It's pretty confusing.